### DISCUSSION TITLE
Why do certain fingerprint fragments disappear as maxPath increases？

### DISCUSSION THREAD

**Discussion by tzyii** [rdkit/rdkit#6752] [Q&A]
```
from rdkit.Chem import AllChem, Draw

mol = AllChem.MolFromSmiles("CC(C)c1ccccc1")

topInfo = dict()
AllChem.RDKFingerprint(mol, maxPath=6, nBitsPerHash=1, branchedPaths=False, fpSize=4096, bitInfo=topInfo)

bits = topInfo.keys()
Draw.DrawRDKitBits([[mol, b, topInfo] for b in bits], legends=[str(b) for b in bits])
```
when maxPath=6, a 1287 bit can be found
![image](https://github.com/rdkit/rdkit/assets/23312659/fac94a2e-4a2a-40d8-bae1-a23cb7994be6)

but when maxPath=7, the 1287 bit disappeared
```
topInfo = dict()
AllChem.RDKFingerprint(mol, maxPath=7, nBitsPerHash=1, branchedPaths=False, fpSize=4096, bitInfo=topInfo)

bits = topInfo.keys()
Draw.DrawRDKitBits([[mol, b, topInfo] for b in bits], legends=[str(b) for b in bits])
```
![image](https://github.com/rdkit/rdkit/assets/23312659/c32583a8-c561-47dd-bb09-591799eb9d01)

I don't understand why the limit on path length has increased, but instead there will be fragments disappearing? Is there a deviation in my understanding of topological fingerprints

**Comment by greglandrum** [comment]
@tzyii : this looks like a bug. Could you please create an issue for it?

**Reply by tzyii** [reply]
Done
